quicknav-1.0.0-win64.spec

```
@@ -0,0 +1,38 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+a = Analysis(
+    ['C:\\Users\\SamLyndon\\Projects\\Personal\\Project-QuickNav-MVP\\quicknav\\cli.py'],
+    pathex=[],
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `pathex`.

(Lint/UselessAssignment)